Source/ablastr/utils/IntervalsParser.cpp

```diff
@@ -0,0 +1,176 @@
+/* Copyright 2022 The ABLASTR Community
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ * Authors: Neil Zaim, Luca Fedeli, Axel Huebl
+ */
+#include "ablastr/utils/IntervalsParser.H"
+
+#include "ablastr/utils/text/StringUtils.H"
+//include "ParserUtils.H"
+#include "Utils/TextMsg.H"
+
+#include <AMReX_Utility.H>
+
+#include <algorithm>
+
+
+ablastr::utils::parser::SliceParser::SliceParser (
+    const std::string& instr,
+    const bool isBTD
```

Comment:
Urgh, such a hack: The `isBTD` should never have made it into this interface and needs to be removed.